test/cluster/services_test.go

```diff
@@ -117,6 +119,36 @@ var _ = Describe(
 				Expect(err).ToNot(HaveOccurred())
 			},
 		)
+
+		It("Validate LB deleted", func() {
+			By("DeployCluster with LB")
+			clusterNamespacedName := getNamespacedName(
+				"load-balancer-delete", namespace,
+			)
+			aeroCluster := createDummyAerospikeCluster(
+				clusterNamespacedName, 2,
+			)
+			aeroCluster.Spec.SeedsFinderServices.LoadBalancer = createLoadBalancer()
+			err := deployCluster(k8sClient, ctx, aeroCluster)
+			Expect(err).ToNot(HaveOccurred())
+
+			By("Validate")
+			validateLoadBalancerExists(aeroCluster)
+
+			By("Delete LB service")
+			aeroCluster, err = getCluster(
+				k8sClient, ctx, clusterNamespacedName,
+			)
+			Expect(err).ToNot(HaveOccurred())
+			aeroCluster.Spec.SeedsFinderServices.LoadBalancer = nil
+			err = updateCluster(k8sClient, ctx, aeroCluster)
+			Expect(err).ToNot(HaveOccurred())
+
+			validateLoadBalancerSvcDeleted(aeroCluster)
+
+			err = deleteCluster(k8sClient, ctx, aeroCluster)
+			Expect(err).ToNot(HaveOccurred())
+		})
```

Comment:
Will need to change this `getCluster` and `deleteCluster` flow once new test changes are merged